Makefile

```diff
@@ -159,8 +135,13 @@ bin-prometheus: mkdir
 #<<<
 bin-traefik: mkdir
 	echo "    started bin-traefik" >> $(LOGFILE)
+ifeq ($(VENV_OS), arm64mac)
+	curl -Lo $(DIR)/bin/miniconda_install.sh \
```

Comment:
the destination path should be /bin/traefik.tar.gz not /bin/miniconda_install.sh

Reply:
thank you for catching that!